code/datums/gamemodes/TDM_tickets.dm

```diff
@@ -0,0 +1,73 @@
+/datum/game_mode/team_death_match
+	name = "Team Death Match"
+	config_tag = "TDM"
+	required_players = 0 //change to 2 in the future
+	votable = TRUE
+
+	var/a_team_tickets = 100
+	var/b_team_tickets = 100
+
+	valid_job_types = list(
+		/datum/job/terragov/command/captain = 1,
+		/datum/job/terragov/command/fieldcommander = 1,
+		/datum/job/terragov/command/staffofficer = 4,
+		/datum/job/terragov/command/pilot = 2,
+		/datum/job/terragov/police/chief = 1,
+		/datum/job/terragov/police/officer = 5,
+		/datum/job/terragov/engineering/chief = 1,
+		/datum/job/terragov/engineering/tech = 1,
+		/datum/job/terragov/requisitions/officer = 1,
+		/datum/job/terragov/medical/professor = 1,
+		/datum/job/terragov/medical/medicalofficer = 6,
+		/datum/job/terragov/medical/researcher = 2,
+		/datum/job/terragov/civilian/liaison = 1,
+		/datum/job/terragov/silicon/synthetic = 1,
+		/datum/job/terragov/silicon/ai = 1,
+		/datum/job/terragov/squad/engineer = 8,
+		/datum/job/terragov/squad/corpsman = 8,
+		/datum/job/terragov/squad/smartgunner = 4,
+		/datum/job/terragov/squad/specialist = 4,
+		/datum/job/terragov/squad/leader = 4,
+		/datum/job/terragov/squad/standard = -1
+	)
+
+//yeah yeah move this to a defines file at some point in the future.
+#define A_TEAM_VICTORY "a_team_victory"
+#define B_TEAM_VICTORY "b_team_victory"
+
+//for when scaling of the round start tickets is important
+///datum/game_mode/team_death_match/proc/scale_tickets()
+//	a_team_tickets = 50 * CONFIG_GET(number/ticket_scale)
+//	b_team_tickets = 50 * CONFIG_GET(number/ticket_scale)
```

Comment:
Make sure you uncomment and implement this if you actually plan on using this feature.